nucliadb/nucliadb/common/maindb/pg.py

```diff
@@ -184,10 +184,11 @@ def __init__(self, pool: asyncpg.Pool, driver: PGDriver):
         self.open = True
 
     async def abort(self):
+        # TODO: Figure out why is this empty?
```

Comment:
@vangheem I see that the abort does nothing for the ReadOnlyPGTransaction.

Is it because we're just fetching a connection from the pool and not using a real PG transaction?